packages/sdk/cryptographic-utils/README.md

```diff
@@ -0,0 +1,7 @@
+# @celo/cryptographic-utils
+
+Here you can find cryptographic functions for
```

Comment:
It would be helpful to provide examples of how to use some of the functions that are available or link to examples.